packages/ethash/README.md

```diff
@@ -0,0 +1,110 @@
+# @ethereumjs/ethash
+
+[![Coverage Status](https://img.shields.io/coveralls/ethereumjs/ethashjs.svg)](https://coveralls.io/r/ethereumjs/ethashjs)
```

Comment:
should we remove this badge now that there is codecov below?

Reply:
Yep! Done.